authors/apps/authentication/templates/email_confirm.html

```diff
@@ -1,6 +1,7 @@
 
     <!doctype html>
     <html>
+<<<<<<< HEAD
```

Comment:
@leonardnjura Why do we still have this?

Reply:
Fixed it. Please check